docs/rest_api.rst

```diff
@@ -19,6 +19,8 @@ For all health check ``GET`` requests Patroni returns a JSON document with the s
 
 - ``GET /replica``: replica health check endpoint. It returns HTTP status code **200** only when the Patroni node is in the state ``running``, the role is ``replica`` and ``noloadbalance`` tag is not set.
 
+- ``GET /replica&lag=<bytes>``: replica check endpoint. In addition to checks from ``replica``, it also checks replication latency and returns status code **200** only when it is below specified value. The key cluster.last_leader_operation from DCS is used for Leader wal position and compute latency on replica for performance reasons.
```

Comment:
I think it has to be `GET /replica?lag=<max-lag>`.

Reply:
If you implement suggestion from https://github.com/zalando/patroni/pull/1599/files#diff-790530624eaa21e5fb614fc321aef277R96 the `max-lag` could be specified as `16777216`, `16777216B`, `16MB`, `1GB`, and so on.